parts/k8s/addons/kubernetesmasteraddons-kube-proxy-daemonset.yaml

```diff
@@ -15,10 +15,9 @@ spec:
         tier: node
     spec:
       tolerations:
-      - key: node-role.kubernetes.io/master
-        operator: Equal
-        value: "true"
-        effect: NoSchedule
+      - key: CriticalAddonsOnly
+        operator: Exists
+      - operator: Exists
```

Comment:
is the double `operator: Exists ` intentional?

Reply:
this is cribbed from kubeadm, believe both are needed.